portfolio/src/main/webapp/index.html

```diff
@@ -5,14 +5,188 @@
+                <p class="photos-dsc">TBT with dad and my glasses!</p>
+                <p class="love-spread">&hearts;</p>
+            </div>
+            <div class="column" style="width:400px">
+                <img src="images/family.jpg" style="width:330px;height:260px;margin-bottom:10px">
+                <p class="photos-dsc">Family portrait skiing!</p>
+                <p class="love-spread">&hearts;</p>
+                <img src="images/zipline.jpg" style="width:330px;height:260px;margin-bottom:10px">
+                <p class="photos-dsc">Zip lining adventure!</p>
+                <p class="love-spread">&hearts;</p>
+            </div>
+        </div>
+        &nbsp;
+        &nbsp;
+        <h2 id="cameraeatsfirst">CAMERA EATS FIRST</h2>
+        <hr style="height:5px;background-color:#6495ED;color:#6495ED;border-width:0;width:400px">
+        <hr style="height:5px;background-color:#d9d5fc;color:#d9d5fc;border-width:0;width:1400px;margin-bottom:40px">
+        <div class="row">
+            <div class="column" style="width:600px">
+                <p class="food-talk">I am definitely not a professional photographer, but I really enjoy taking pictures, specifically of food. I actually enjoy it so much that I can’t never start eating before taking a snapshot of it first. 
+                    People usually complain, but I always remind them that the camera eats first! Until now, I had  never thought about why I do it, but now I realize that there are mainly 2 motives behind it.</p>
+                <img src="images/fish.jpg" style="width:370px;height:260px;margin-bottom:35px;margin-top:20px">
+                <p class="food-talk">The first one is that I am very oddly satisfied by well-thought presentations of plates, and the way they’re placed in a manner appealing to the consumer, in this case, me! Here I share with you 3 examples of food-shots taken by me. 
+                    The top right portrays a typical breakfast in my home country: an “arepa” with multiple sides, scrambled eggs, cheese, ham, and of course, some shredded beef and a touch of black beans. To the left, it’s a meal for a beach day: fried fish, rice and some fresh salad. 
+                    Below, I bet you know it: mini-burgers and french fries!</p>
+            </div>
+            <div class="column" style="width:600px">
+                <img src="images/arepa.jpg" style="width:370px;height:260px;margin-bottom:20px">
+                <p class="food-talk">The other purpose, well, it’s just a matter of initializing small talk with friends. Sometimes you’re trying with all your efforts to maintain a healthy diet and sharing what you eat with others on the same plan, makes you feel supported and gives you more motivation.
+                     Occasionally, you feel like cooking famous recipes from scratch and by your own, so why not share it. I am the kind of person that cuts the chocolate lava-cake in half just to record it, be prepared!</p>
```

Comment:
Maybe "why not share it?" because it's a question and "just to record it so be prepared!"

Reply:
Fixed!